config/config.json

```diff
@@ -40,7 +31,7 @@
     }
   },
   "logging": {
-    "level": "info",
+    "level": "debug",
```

Comment:
nit; not sure if you want to revert this back to `info`?

Reply:
Thanks for noticing. I will tackle improvements in upcoming PRs. This whole file will be renamed to config.example.json